virtcontainers/network.go

```diff
@@ -73,12 +61,8 @@ func (n NetInterworkingModel) IsValid() bool {
 const (
 	defaultNetModelStr = "default"
 
-	bridgedNetModelStr = "bridged"
-
```

Comment:
Won't this require changes to the config files (`internetworking_model` comments)?:

- https://github.com/kata-containers/runtime/blob/master/cli/config/configuration-fc.toml.in
- https://github.com/kata-containers/runtime/blob/master/cli/config/configuration-qemu.toml.in

Reply:
@jodh-intel I missed that. Let me fix it

Reply:
fixed